### PR TITLE
Create cdn.aayush

### PR DESCRIPTION
**Pull request by OpAayush**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please rename your file to: `cdn.aayush.json` and add an homepage.

**Comment by OpAayush** [comment]
yo accept

File: domains/cdn.aayush.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "CDN for Aayu5h",
+  "repo": "https://github.com/OpAayush/cdn",
+  "owner": {
+    "username": "OpAayush",
+    "email": "aayush.agrawalm@gmail.com"
+  },
+  "record": {
+    "CNAME": "opaayush.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/cdn.aayush.json:9]: Please add a homepage.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by OpAayush** [comment]
Can you merge?

**Comment by OpAayush** [comment]
but its actual website is aayush.is-a.dev who has a homepage for cdn bro...

**Comment by OpAayush** [comment]
Kk

**Comment by OpAayush** [comment]
what about now? accept?

**Review by wdhdev — APPROVED** [review]

**Review by andrewstech — APPROVED** [review]

**Pull request merged**